src/emulator/hub.ts

```diff
@@ -64,9 +64,7 @@ export class EmulatorHub extends ExpressBasedEmulator {
   }
 
   static getLocatorFilePath(projectId: string): string {
-    const dir = os.tmpdir();
-    const filename = `hub-${projectId}.json`;
-    return path.join(dir, filename);
+    return path.join(os.homedir(), ".cache", "firebase", `hub-${projectId}.json`);
```

Comment:
What if the parent directories do not exist? Can we find some existing code in the rest of the CLI code base that creates the parents and reuse the logic?